README.md

```diff
@@ -360,24 +356,24 @@ docker stack rm caddy-docker-demo
 ### With run commands
 
 ```
-docker run --name caddy -d -p 2015:2015 -v /var/run/docker.sock:/var/run/docker.sock lucaslorentz/caddy-docker-proxy:ci-alpine -agree -email email@example.com -log stdout -docker-process-caddyfile
+docker run --name caddy -d -p 443:443 -v /var/run/docker.sock:/var/run/docker.sock lucaslorentz/caddy-docker-proxy:ci-alpine docker-proxy
 
-docker run --name whoami0 -d -l caddy.address=whoami0.example.com -l caddy.targetport=8000 -l caddy.tls=off jwilder/whoami
+docker run --name whoami0 -d -l caddy.address=whoami0.example.com -l caddy.targetport=8000 -l caddy.tls=internal jwilder/whoami
 
-docker run --name whoami1 -d -l caddy.address=whoami1.example.com -l caddy.targetport=8000 -l caddy.tls=off jwilder/whoami
+docker run --name whoami1 -d -l caddy.address=whoami1.example.com -l caddy.targetport=8000 -l caddy.tls=internal jwilder/whoami
 
-curl -H Host:whoami0.example.com http://localhost:2015
-curl -H Host:whoami1.example.com http://localhost:2015
+curl -k --resolve whoami0.example.com:443:127.0.0.1 https://whoami0.example.com
+curl -k --resolve whoami1.example.com:443:127.0.0.1 https://whoami1.example.com
 
 docker rm -f caddy whoami0 whoami1
 ```
 
 ## Building it
-You can use our caddy build wrapper **build.sh** and include additional plugins on https://github.com/lucaslorentz/caddy-docker-proxy/blob/master/main.go#L5
+You can use our caddy build wrapper **build.sh** and include additional plugins on https://github.com/lucaslorentz/caddy-docker-proxy/blob/master/main.go#L8
```

Comment:
I think you could point people to use https://github.com/caddyserver/xcaddy instead. There's some docs on the site for it:

https://caddyserver.com/docs/build#xcaddy

Could do the building that way in your Docker image possibly as well (multi-stage, first stage is Go plus xcaddy, then 2nd stage you just copy the binary)

Reply:
Seconding this; xcaddy is probably the best way to make custom Caddy builds.

Reply:
https://github.com/lucaslorentz/caddy-docker-proxy/issues/141